src/app/functions/server/dashboard.test.ts

```diff
@@ -798,4 +799,494 @@ describe("getDashboardSummary", () => {
+            inProgressDataPointCounts,
+            "scan-result",
+          );
+          const dataBrokerManuallyResolvedNum = countTotalExposures(
+            manuallyResolvedDataPointCounts,
+            "scan-result",
+          );
+          const dataBrokerManuallyResolvedDataPointsNum = countTotalDataPoints(
+            manuallyResolvedDataPointCounts,
+            "scan-result",
+          );
+          const dataBrokerUnresolvedNum = countTotalExposures(
+            unresolvedDataPointCounts,
+            "scan-result",
+          );
+          const dataBrokerUnresolvedDataPointsNum = countTotalDataPoints(
+            unresolvedDataPointCounts,
+            "scan-result",
+          );
+          const dataBrokerTotalNum =
+            dataBrokerUnresolvedNum +
+            dataBrokerAutoFixedNum +
+            dataBrokerInProgressNum +
+            dataBrokerManuallyResolvedNum;
+          const dataBrokerTotalDataPointsNum =
+            dataBrokerUnresolvedDataPointsNum +
+            dataBrokerAutoFixedDataPointsNum +
+            dataBrokerInProgressDataPointsNum +
+            dataBrokerManuallyResolvedDataPointsNum;
+          const totalDataPointsNum =
+            dataBreachTotalDataPointsNum + dataBrokerTotalDataPointsNum;
+
+          const expectedSummary: Omit<
+            DashboardSummary,
+            "fixedSanitizedDataPoints" | "unresolvedSanitizedDataPoints"
+          > = {
+            allDataPoints: allDataPointCounts,
+            unresolvedDataPoints: unresolvedDataPointCounts,
+            inProgressDataPoints: inProgressDataPointCounts,
+            manuallyResolvedDataBrokerDataPoints:
+              manuallyResolvedDataPointCounts,
+            fixedDataPoints: resolvedDataPointCounts,
+            dataBreachTotalNum: dataBreachTotalNum,
+            dataBreachResolvedNum: dataBreachResolvedNum,
+            dataBreachUnresolvedNum: dataBreachUnresolvedNum,
+            dataBreachTotalDataPointsNum: dataBreachTotalDataPointsNum,
+            dataBreachFixedDataPointsNum: dataBreachFixedDataPointsNum,
+            dataBrokerTotalNum: dataBrokerTotalNum,
+            dataBrokerTotalDataPointsNum: dataBrokerTotalDataPointsNum,
+            dataBrokerAutoFixedNum: dataBrokerAutoFixedNum,
+            dataBrokerAutoFixedDataPointsNum: dataBrokerAutoFixedDataPointsNum,
+            dataBrokerInProgressNum: dataBrokerInProgressNum,
+            dataBrokerInProgressDataPointsNum:
+              dataBrokerInProgressDataPointsNum,
+            dataBrokerManuallyResolvedNum: dataBrokerManuallyResolvedNum,
+            dataBrokerManuallyResolvedDataPointsNum:
+              dataBrokerManuallyResolvedDataPointsNum,
+            totalDataPointsNum: totalDataPointsNum,
+            // TODO: Figure out what these should be and, when we do, replace
+            //       `toMatchObject` by `toStrictEqual`:
+            // unresolvedSanitizedDataPoints: [],
+            // fixedSanitizedDataPoints: [],
```

Comment:
Note that I couldn't decipher the santisation code yet, so I left that to future work. So there might still be uncovered bugs there 🙂